RJS3001/a1.js

```diff
@@ -0,0 +1,94 @@
+// Bài 1: Viết hàm
+// -	Input: Cho vào chuỗi bất kì
+// -	Output: Đếm xem chuỗi đó có bao nhiêu nguyên âm (Nguyên âm là a, e, I, o, u)
+// Bài 2:  Viết hàm
+// -	Input: Cho 1 mảng
+// -	Output: Tìm số lần xuất hiện của các phần tử trong mảng
+// Bài 3: Viết hàm
+// -	Input: Cho 1 chuỗi
+// -	Output: In ra chuỗi đảo ngược
+// Bài 4:
+// - Input: cho chuỗi có định dạng cách nhau bởi _ ví dụ: user_name. 
+// - Output: Viết hàm để in ra viết hoa các kí tự đầu tiên và bỏ dấu _: ví dụ như user_name => UserName
+// Bài 5: Viết function để đổi chỗ 2 số
+
+// * Bai 1 * //
+var string = "kate do is a pink bunny";
+function vowelCount(str)
+{
+var count = 0;
+for (let i=0; i <= str.length-1; i++ )
+{
+    if (['a', 'e', 'i', 'o', 'u'].includes(str.charAt(i)))
+    {
+        ++count;
+    }
+}
+return console.log(`Number of vowels in string "${string}" is: ${count}`);
+};
+vowelCount(string);
+
+// * Bai 2 * //
+var array = [1,2,5,7,1,1,5,2,7,9];
+function count_frequency(input)
+{
+    var i = 0;
+    while (i<=input.length-1)
+    {
+        if (!input.slice(0,i).includes(input[i]))
+        {
+            var n=1;
```

Comment:
- Bài này giải theo cách này cũng oke nhưng độ phức tạp của nó là O(n^n) nên chỗ này để ý độ phức tạp của thuật toán
Vì ở đây là loop lồng loop nên độ phức tạp là O(n^n)

Reply:
done